src/app/layout.tsx

```diff
@@ -1,28 +1,37 @@
+"use client";
 import type { Metadata } from "next";
 import "@/styles/globals.css";
 import { Inter as FontSans } from "next/font/google";
 import { cn } from "@/lib/utils";
+import { useState } from "react";
+import React from "react";
+import { ClerkProvider } from "@clerk/nextjs";
 import Header from "@/components/header";
 import Footer from "@/components/footer";
-import { ClerkProvider } from "@clerk/nextjs";
 import Sidebar from "@/components/sidebar";
 
 export const fontSans = FontSans({
   subsets: ["latin"],
   variable: "--font-sans",
 });
 
-//! Update metadata to match your project
-export const metadata: Metadata = {
-  title: "Ecologistics",
-  description: "Ecologistics Web Portal",
-};
+// may need to be moved elsewhere
+// export const metadata: Metadata = {
+//   title: "Ecologistics",
+//   description: "Ecologistics Web Portal",
+// };
```

Comment:
this object defines what shows up in the website title and description, so we shouldn't need to change it

Reply:
![image](https://github.com/hack4impact-calpoly/ecologistics-portal/assets/76940774/7de8e411-e6cd-419b-bd9c-33a7414f0f87)
I run into this compile error when I leave in the metadata. Not sure if this is an issue just for me, feel free to experiment!